css/index.css

```diff
@@ -111,6 +132,38 @@ p {
     align-items: center;
     justify-content: center;
 }
+/* coloring boxes */
```

Comment:
Nice work letting the parent style the individual boxes and only overriding the color in each box.